src/main/java/net/rptools/maptool/model/sheet/stats/StatSheetContext.java

```diff
@@ -0,0 +1,338 @@
+                  }
+                }
+                properties.add(new Property(tp.getName(), tp.getShortName(), value, tp.isGMOnly()));
+              }
+            });
+
+    Dimension dim;
+    if (token.getPortraitImage() != null) {
+      var image = ImageManager.getImage(token.getPortraitImage());
+      dim = new Dimension(image.getWidth(), image.getHeight());
+    } else {
+      var image = ImageManager.getImage(token.getImageAssetId());
+      dim = new Dimension(image.getWidth(), image.getHeight());
+    }
+    SwingUtil.constrainTo(dim, AppPreferences.getPortraitSize());
+    portraitWidth = dim.width;
+    portraitHeight = dim.height;
+
+    statSheetLocation =
+        switch (location) {
+          case TOP_LEFT -> "statSheet-topLeft";
+          case TOP_RIGHT -> "statSheet-topRight";
+          case BOTTOM_LEFT -> "statSheet-bottomLeft";
+          case BOTTOM_RIGHT -> "statSheet-bottomRight";
+          case TOP -> "statSheet-top";
+          case BOTTOM -> "statSheet-bottom";
+          case LEFT -> "statSheet-left";
+          case RIGHT -> "statSheet-right";
+        };
+  }
+
+  /**
+   * Returns the name of the token.
+   *
+   * @return The name of the token.
+   */
+  public String getName() {
+    return name;
+  }
+
+  /**
+   * Returns the GM name of the token.
+   *
+   * @return The GM name of the token.
+   */
+  public String getGmName() {
+    return gmName;
+  }
+
+  /**
+   * Returns the image asset of the token.
+   *
+   * @return The image asset of the token.
+   */
+  public String getImage() {
+    return imageAsset != null ? "asset://" + imageAsset : null;
+  }
+
+  /**
+   * Returns the portrait asset of the token.
+   *
+   * @return The portrait asset of the token.
+   */
+  public String getPortrait() {
+    return portraitAsset != null ? "asset://" + portraitAsset : null;
+  }
+
+  /**
+   * Returns the label of the token.
+   *
+   * @return The label of the token.
+   */
+  public String getLabel() {
+    return label;
+  }
+
+  /**
+   * Returns the width of the portrait on the stat sheet.
+   *
+   * @return The width of the portrait on the stat sheet.
+   */
+  public int getPortraitWidth() {
+    return portraitWidth;
+  }
+
+  /**
+   * Returns the height of the portrait on the stat sheet.
+   *
+   * @return The height of the portrait on the stat sheet.
+   */
+  public int getPortraitHeight() {
+    return portraitHeight;
+  }
+
+  /**
+   * Returns the properties of the token.
+   *
+   * @return The properties of the token.
+   */
+  public List<Property> getProperties() {
+    return properties;
+  }
+
+  /**
+   * Returns the css class for the location of the stat sheet.
+   *
+   * @return The css class for the location of the stat sheet.
+   */
+  public String getStatSheetLocation() {
+    return statSheetLocation;
+  }
+
+  /**
+   * Returns the notes of the token.
+   *
+   * @return The notes of the token.
+   */
+  public String getNotes() {
+    return notes;
```

Comment:
Shouldn't we StringUtil.htmlize the noted to account for the different possible note types possible?

Reply:
Updated getNotes and getGMNotes to return HTML